platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/sensors/ButtonGenerator.xtend

```diff
@@ -49,10 +49,10 @@ class ButtonGenerator extends AbstractSystemResourceGenerator {
             void «handler.internalHandlerName»(uint32_t data)
             {
             	if(data == BSP_XDK_BUTTON_PRESS) {
-            		Retcode_T retcode = CmdProcessor_enqueueFromIsr(&Mita_EventQueue, «handler.handlerName», NULL, data);
+            		Retcode_T retcode = CmdProcessor_EnqueueFromIsr(&Mita_EventQueue, «handler.handlerName», NULL, data);
```

Comment:
Are you using an old SDK? the button generator doesn't seem relevant to BLE.

Reply:
yeah , it is not related to BLE but warnings were coming from this folder